x-pack/packages/kbn-ai-assistant/src/hooks/use_once.ts

```diff
@@ -10,6 +10,10 @@ import { useRef } from 'react';
 export function useOnce<T>(variable: T): T {
   const ref = useRef(variable);
 
+  if (!ref.current && variable) {
+    ref.current = variable;
```

Comment:
NOoO!!! this goes against the purpose of this hook. Why is this needed?

Reply:
Reverted in https://github.com/elastic/kibana/pull/194552/commits/911c57e4768d24833c236d942d6c860f4fed41f9